poly_borders/poly_borders_tool/poly_borders_tool.cpp

```diff
@@ -0,0 +1,64 @@
+#include "poly_borders/borders_data.hpp"
+#include "poly_borders/help_structures.hpp"
+
+#include "platform/platform.hpp"
+
+#include "base/assert.hpp"
+#include "base/exception.hpp"
+#include "base/logging.hpp"
+
+#include <exception>
+
+#include "3party/gflags/src/gflags/gflags.h"
+
+DEFINE_string(borders_path, "", "Path to directory with *.poly files.");
+DEFINE_string(output_path, "", "Path to target directory where the output *.poly files will be placed.");
+
+using namespace poly_borders;
+
+int main(int argc, char ** argv)
+{
+  google::ParseCommandLineFlags(&argc, &argv, true);
+  google::SetUsageMessage("\n\n\tThe tool is used to process *.poly borders files. We use such files\n"
+                          "\tfor \"planet to mwms separation\" in generator. The problem is that we have\n"
```

Comment:
s/to mwms separation/to separate mwms/
(or "to cut the planet into mwms", or something similar)

Reply:
We use such files to cut the planet into mwms in generator.